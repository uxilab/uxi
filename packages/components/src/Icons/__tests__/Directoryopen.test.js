import React from 'react';
import Directoryopen from '../Directoryopen';

describe('Icon : <Directoryopen />', () => {
  it('match snapshot', () => {
    expect(shallow(<Directoryopen />)).toMatchSnapshot();
  });
});

