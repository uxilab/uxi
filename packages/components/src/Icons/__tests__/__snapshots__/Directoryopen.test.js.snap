// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Directoryopen /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 19.26 13.84"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M19,3a1.41,1.41,0,0,0-1.16-.61H7.91L6.6.66A1.65,1.65,0,0,0,5.29,0H1.64A1.65,1.65,0,0,0,0,1.64V12.18a1.64,1.64,0,0,0,1.63,1.64h0l12.87,0a1.4,1.4,0,0,0,1.31-.9l3.35-8.61A1.42,1.42,0,0,0,19,3ZM5,2.4a1.42,1.42,0,0,0-1.32.93l-2,5.49V1.67H5.28l.55.74ZM17.65,3.94l-3.19,8.21H2.1l3-8.2Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
